### PR TITLE
minor weaks to github actions

### PR DESCRIPTION
**Pull request by auvipy**

### CODE REVIEW

**Comment by auvipy** [comment]
https://github.com/eventlet/eventlet/issues/730

File: requirements/extras/eventlet.txt [hunk 1]
```diff
@@ -1 +1 @@
-eventlet>=0.26.1; python_version<"3.10"
+eventlet>=0.32.0; python_version<"3.11"
```
Comment (auvipy) [inline requirements/extras/eventlet.txt:1]: miggt end up pin <3.10 for now

**Comment by auvipy** [comment]
ERROR:   pypy3-unit: could not install deps [-rD:\a\celery\celery/requirements/default.txt, -rD:\a\celery\celery/requirements/test.txt, -rD:\a\celery\celery/requirements/pkgutils.txt, -rD:\a\celery\celery/requirements/test-ci-default.txt]; v = InvocationError("'D:\\a\\celery\\celery\\.tox\\pypy3-unit\\Scripts\\python.EXE' -m pip install '-rD:\\a\\celery\\celery/requirements/default.txt' '-rD:\\a\\celery\\celery/requirements/test.txt' '-rD:\\a\\celery\\celery/requirements/pkgutils.txt' '-rD:\\a\\celery\\celery/requirements/test-ci-default.txt'", 1)

**Comment by auvipy** [comment]
@matusvalo any clue? pypy on windows failing, though I saw it passing

**Comment by matusvalo** [comment]
pypy-3.7 is randomly crashing on windows so it is just matter of rerunning the jobs. pypy-3.8 is broken on windows not sure why.

**Comment by auvipy** [comment]
well that's eco system issue i think. I am not sure we are getting pypy 7.3.6 on github action as of now. see https://github.com/actions/setup-python/commit/002b1e40449cd70913538589c187c2728a41cb61

**Comment by auvipy** [comment]
pypy3.8 issue will  be fixed eventually. merging after lots of wastage of energy

**Pull request merged**